public/discussion_list.php

```diff
@@ -7,18 +7,16 @@
         <table>
             <thead>
                 <tr>
-                    <th>#</th>
                     <th>Title</th>
-                    <th>User ID</th>
+                    <th>Author</th>
                     <th>Created At</th>
                 </tr>
             </thead>
             <tbody>
         <?php foreach ($discussions as $row) { ?>
             <tr>
-                <td><?php echo escape($row->id); ?></td>
                 <td><a href="discussion.php?id=<?php echo $row->id; ?>"><?php echo escape($row->title); ?></a></td>
-                <td><?php echo escape($row->user_id); ?></td>
+                <td><?php echo escape($row->user->first_name." ".$row->user->last_name); ?></td>
```

Comment:
hey didnt we implement  get_full_name()

Reply:
oh that's right I implemented that! lol will change it

Reply:
done